File: src/log.js
// Really simple wrapper over console.log - I hate console.log bs

const config = require("../config.json");
let debug = config.debug ? config.debug : 0;

function isDebug(level){
    return debug >= level;
}

function l(msg, level){
    if(!level || debug >= level){
        console.log(msg);
    }
}

function w(msg,level){
    if(!level || debug >= level){
        console.log("[WARNING] " + msg);
    }
}

function e(msg,level){
    if(!level || debug >= level){
        console.log("[ERROR] " + msg);
    }
}

function i(msg,level){
    if(!level || debug >= level){
        console.log("[INFO] " + msg);
    }
}

module.exports = {
    l, w, e, isDebug
}

File: src/log.py
# Really simple wrapper over console.log - I hate console.log bs
import json
config = None
### Extracting config
with open('./config.json', 'r') as fconfig: # TODO handle error
    config = json.loads(fconfig.read())
    
debug = config['debug'] if config['debug'] else 0

class Log:
    @staticmethod
    def isDebug(l):
        return (debug >= l) if l else False

    @staticmethod
    def l(msg,level=0):
        if debug >= level:
            print(msg)
    @staticmethod
    def w(msg,level=0):
        if debug >= level:
            print(msg)
    @staticmethod
    def w(msg,level=0):
        if debug >= level:
            print(msg)

